include/bitcoin/bitcoin/machine/rule_fork.hpp

```diff
@@ -73,6 +73,9 @@ enum rule_fork : uint32_t
     /// Fix Satoshi's time warp bug (hard fork, feature).
```

Comment:
I think this qualifies as "security" vs. "feature".

Reply:
Done.